<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLineMapEdit</class>
 <widget class="QDialog" name="QLineMapEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Route Map Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,0,0">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1137</width>
        <height>262</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QGLESLineMapCanvas" name="openGLWidget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,2">
     <item>
      <widget class="QGroupBox" name="gbBasicMenu">
       <property name="title">
        <string>Property</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editTitle"/>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Tile Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editTileSize"/>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Line Thickness</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editLineThick"/>
          </item>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editWidth"/>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editHeight"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="m_grpEditMode">
         <property name="title">
          <string>Edit Mode</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="m_ctlTile">
            <property name="text">
             <string>Tile (Line)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="m_ctlArrow">
            <property name="text">
             <string>Arrow (No Use)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="m_ctlDisplayItem">
            <property name="text">
             <string>Display Pool</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="m_ctlNode">
            <property name="text">
             <string>Node (Station Name / Icon)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="m_ctlImageList">
            <property name="text">
             <string>Image List (No Use)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Colour</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Background</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QColorPickerPushButton" name="m_ctlBackgroundColor">
              <property name="text">
               <string>Change Colour</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Line</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QColorPickerPushButton" name="m_ctlForegroundColor">
              <property name="text">
               <string>Change Colour</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QPushButton" name="m_pbApplyColor">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="m_ctlStackedWidget">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QColorPickerPushButton" name="m_ctlPassedColor">
             <property name="text">
              <string>Passed Station Color</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Passed Station Color</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QColorPickerPushButton" name="m_ctlFocusedColor">
             <property name="text">
              <string>Focused Station Color</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Focused Station Color</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QColorPickerPushButton" name="m_ctlLeftColor">
             <property name="text">
              <string>Remained Station Color</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Remained Station Color</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QColorPickerPushButton" name="m_ctlArrowPassedColor">
             <property name="text">
              <string>Arrow Color for Passed Area</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Arrow Color for Passed Area</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QColorPickerPushButton" name="m_ctlArrowFocusedColor">
             <property name="text">
              <string>Arrow Color for Focused Area</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Arrow Color for Focused Area</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QColorPickerPushButton" name="m_ctlArrowSecondFocusedColor">
             <property name="text">
              <string>Second Arrow Color for Focused Area</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Second Arrow Color for Focused Area</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QColorPickerPushButton" name="m_ctlArrowRemainedColor">
             <property name="text">
              <string>Arrow Color for Remained area</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Arrow Color for Remained area</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageTileEdit">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Line (Link)</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Spare</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>Spare</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>Spare</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="m_tblLineMapLink"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageNodeEdit">
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="1">
            <widget class="QPushButton" name="btnPasteNode">
             <property name="text">
              <string>Paste Node Name Property</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="btnPasteSpot">
             <property name="text">
              <string>Paste Node Icon Property</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btnShowSpotDefault">
             <property name="text">
              <string>Set All Nodes Icon Default</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btnShowSpotCustom">
             <property name="text">
              <string>Set All Nodes Icon Custom</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="btnCopyProp">
             <property name="text">
              <string>Copy Selected Node Property</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QTableView" name="m_tblLineMapNode"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageArrowEdit">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Forward </string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="m_spForwardArrowOffset">
                 <property name="minimumSize">
                  <size>
                   <width>110</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="value">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Backward</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="m_spBackwardArrowOffset">
                 <property name="minimumSize">
                  <size>
                   <width>110</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="value">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Width</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSpinBox" name="m_spArrowPenWidth">
                 <property name="minimumSize">
                  <size>
                   <width>110</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="value">
                  <number>10</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="m_pbApplyArrowOffset">
               <property name="text">
                <string>Apply</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>112</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageDisplayEdit">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
           <item>
            <widget class="QTableView" name="m_tblDisplayItemPool"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageImageList">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QTableView" name="m_tblImageIndexList"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QColorPickerPushButton</class>
   <extends>QPushButton</extends>
   <header>qcolorpickerpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QGLESLineMapCanvas</class>
   <extends>QOpenGLWidget</extends>
   <header>QGLESLineMapCanvas.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
